### PR TITLE
Update to hosted payment simulator

### PR DESCRIPTION
**Pull request by andrewpaliga**
We'll need to rename the url and a few other changes if we are good with the name change. 

@girasquid @louiskearns @mhashemi86

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,132 +1,21 @@
-Shopify Offsite Gateway Simulator
+Hosted Payment SDK Simulator
 ===========================
 
-This is a common API that simplifies the onboarding of new payment providers, specifically Offsite Gateways / Hosted Payment Pages, that are looking to be used on [Shopify](http://www.shopify.com). By supporting a common API for redirect, cancel, complete, and callback phases of a payment flow, we are giving gateway implementers a way to integrate with Shopify at any time, without requiring a large amount of customization or validation work or bottlenecks by the Shopify integration team.
+This tool will allow you to simulate the redirects and callbacks needed to build and integration using the [Hosted Payment SDK](https://docs.shopify.com/hosted-payment-sdk). It also serves as a [calculator](https://offsite-gateway-sim.herokuapp.com/calculator) that can be used to verify your signature algorithm.
```
Comment (girasquid) [inline README.md]: "an" integration

File: README.md [hunk 2]
```diff
@@ -1,132 +1,21 @@
...
 
-### Getting Started
+To use the simulator please familiarize yourself with the [Hosted Payment SDK](https://docs.shopify.com/hosted-payment-sdk) documentation and then:
 
-Follow these simple steps to get started.
+1. Add a payment gateway with “Redirect URL” of `https://offsite-gateway-sim.herokuapp.com/`.
```
Comment (girasquid) [inline README.md]: Change smart quotes to regular quotes

File: README.md [hunk 3]
```diff
@@ -1,132 +1,21 @@
...
 
-1. [Sign in](http://www.shopify.com/login) to your Shopify store.
-2. Go to [Products](http://www.shopify.com/admin/products) and [add a dummy product](http://docs.shopify.com/manual/your-store/products/create-product).
-2. Go to [Settings/Payments](http://www.shopify.com/admin/settings/payments), and select the **Universal Offsite Dev Kit** in the gateway dropdown.
-3. Complete the 3 fields,
-  + ``x_account_id`` - this is an identifier for a test merchant on your system
-  + ``HMAC key`` - this is a key your gateway will use to verify requests and sign responses
-  + ``POST URL`` - this is URL on your system that will properly handle [Request Values](#request-values) and is then able to provide proper [Response Values](#response-values) to various return URLs at Shopify
-4. Now you're ready to test! From your admin, click the 'view your website' link, and add a product to your cart. On the cart page, click the "Check out" button, enter in some dummy info, and complete the checkout using your gateway.
+3. Complete a test purchase on your shop (you may need to add a product first). At the end of checkout you will be redirect to the sceen below.
```
Comment (girasquid) [inline README.md]: "redirectED"

File: README.md [hunk 4]
```diff
@@ -1,132 +1,21 @@
...
-| ``x_gateway_reference`` | unicode string                                           | ✓         | 123                                      | Unique reference for the authorization issued by the payment processor. |
-| ``x_timestamp``         | [iso-8601](http://en.wikipedia.org/wiki/ISO_8601) in UTC | ✓         | 2014-03-24T12:15:41Z                     | Time of transaction completion.<br>UTC Time: YYYY-MM-DDTHH:MM:SSZ                                                                        |
-| ``x_result``            | fixed choice                                             | ✓         | completed                                | One of: completed, failed, pending                                      |
-| ``x_signature``         | hex string, case-insensitive                             | ✓         | 49d3166063b4d881b50af0b4648c1244bfa9890a53ed6bce6d2386404b610777 | See [Signing Mechanism](#signing-mechanism).                            |
+Please email paymment-integrations@shopify.com if you have any questions.
```
Comment (girasquid) [inline README.md]: `payment-integrations@shopify.com`

**Comment by girasquid** [comment]
Could use a squash

File: README.md (hunk 2 above)
Comment (andrewpaliga) [inline README.md]: Don't we want smart quotes?

**Comment by andrewpaliga** [comment]
Made a few more changes

File: README.md (hunk 2 above)
Comment (girasquid) [inline README.md]: If you do, then make the other quotes in this file consistent

**Pull request merged**